main.c

```diff
@@ -33,9 +33,9 @@ static uint32_t parse_color(const char *color) {
 
 	int len = strlen(color);
 	if (len != 6 && len != 8) {
-		swaylock_log(LOG_DEBUG, "Invalid color %s, defaulting to 0xFFFFFFFF",
+		swaylock_log(LOG_DEBUG, "Invalid color %s, defaulting to 0xA3A3A3FF",
 				color);
-		return 0xFFFFFFFF;
+		return 0xA3A3A3FF;
```

Comment:
This applies to all colors, not just the background. I'd prefer to leave this as-is.